src/v/config/configuration.cc

```diff
@@ -1118,6 +1118,15 @@ configuration::configuration()
       "Interval for cloud storage housekeeping tasks",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       5min)
+  , cloud_storage_max_segments_pending_deletion(
+      *this,
+      "cloud_storage_max_segments_pending_deletion",
+      "The limit for the number of segments pending deletion from the cloud."
```

Comment:
This is per-partition, right?  Should either explain that in the description or make the name end _per_partition (which I'm conscious makes it very long...)

Reply:
Good point. Updated the name and the description.